cli/schema/cypress.schema.json

```diff
@@ -124,14 +124,6 @@
           "default": false,
           "description": "Path to folder containing component test files (Pass false to disable)"
         },
-        "pluginsFile": {
-          "type": [
-            "string",
-            "boolean"
-          ],
-          "default": "cypress/plugins/index.js",
-          "description": "Path to plugins file. (Pass false to disable)"
-        },
```

Comment:
This file could be happily deleted.

Reply:
Leaving this work for https://cypress-io.atlassian.net/browse/UNIFY-1291 
We have a system test that pulls this schema from  https://raw.githubusercontent.com/cypress-io/cypress/develop/cli/schema/cypress.schema.json which will need to be removed and handled correctly.